elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -26,7 +26,6 @@ $LOCAL-VARIABLES: (
   TextDecoration--hover: none,
   TextDecoration--focus: none,
   FontWeight: pfe-var(font-weight--bold),
-  FontFamily: pfe-var(font-family--heading),
```

Comment:
@starryeyez024 Don't you need to remove the styles from the `::slotted(*)` selector below as well?

Reply:
Hmm.. there could still be value in having that local variable around, in case we start moving more stuff into the shadow DOM. For instance this works today:

```css
body {
    font-family: "Comic Sans MS";
}
pfe-cta {
    font-family: "Trebuchet"; 
}
```
but if the CTA link (or the whole CTA was moved to the shadow DOM, like in the case of pfe-navigation, you'd need this in order update the font family:

```css
body {
    font-family: "Comic Sans MS";
}
pfe-cta { 
    --pfe-cta--FontFamily: "Trebuchet";
}
```

Is this overkill?

Reply:
@castastrophe  ^^

Reply:
font-family pierces the shadow dom